src/app/shared/menu-items/menu-items.ts

```diff
@@ -7,7 +7,7 @@ export interface Menu {
   icon: string;
 }
 
-const MENU_ITEMS: Menu[] = [
+const MENUITEMS = [
```

Comment:
Looks like some changes from the main were accidentally replaced with older code

Reply:
+